posts/2022-09-01-openliberty-guides-in-japanese.adoc

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "OpenLibertyのガイドを使ってクラウドネイティブのJavaアプリケーション開発とKubenetes環境へのデプロイメントについて学ぼう"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/una-tapa
+author_github: https://github.com/una-tapa
+seo-title: OpenLibertyのガイドを使ってクラウドネイティブのJavaアプリケーション開発とKubenetes環境へのデプロイメントについて学ぼう - OpenLiberty.io
+seo-description: この投稿では、OpenLibertyのガイドの概要を説明します。OpenLibertyのガイドを使うと、JakartaEEとmicroprofileを使ったクラウドネイティブのJavaアプリケーション開発とKubenetes環境へのデプロイメントを効率的に学ぶことができます。
+blog_description: "この投稿では、OpenLibertyのガイドの概要を説明します。OpenLibertyのガイドを使うと、JakartaEEとmicroprofileを使ったクラウドネイティブのJavaアプリケーション開発とKubenetes環境へのデプロイメントを効率的に学ぶことができます。"
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+additional_authors:
+- name: 田中孝清
+  github: https://github.com/takakiyo
+  image: https://avatars0.githubusercontent.com/takakiyo
+---
+= OpenLibertyのガイドを使ってクラウドネイティブのJavaアプリケーション開発とKubenetes環境へのデプロイメントについて学ぼう
+
+高宮裕子 <https://github.com/una-tapa> 田中孝清 <https://github.com/takakiyo>
```

Comment:
```suggestion
高宮裕子 <https://github.com/una-tapa>
```

In my testing, the newline needs to be removed and the second author is not needed on this line because that author is already defined in `additional_authors:`